Makefile

```diff
@@ -20,7 +19,14 @@ generate:
 run:
 	swift build
 	.build/debug/tuist $(ARGS)
+lint-fix:
```

Comment:
@kwridan I'll merge [this one](https://github.com/tuist/tuist/pull/5451) first and then push a commit to this one extracting the logic into their own scripts.